target/linux/ramips/image/mt7621.mk

```diff
@@ -719,6 +723,25 @@ define Device/lenovo_newifi-d1
 endef
 TARGET_DEVICES += lenovo_newifi-d1
 
+define Device/linksys_e5600
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 26624k
+  DEVICE_VENDOR := Linksys
+  DEVICE_MODEL := E5600
+  DEVICE_PACKAGES := kmod-mt7603 kmod-mt7615e kmod-mt7615-firmware \
```

Comment:
```kmod-mt7615-firmware``` is for MT7615 chip, not for MT7613 (MT7663).

Reply:
As per following topic discussed, it requires kmod-mt7615e and kmod-mt7663-firmware-ap. I checked with other options but unless we have kmod-mt7515e, 5GHz wifi does not work at all.
The topic link is:
https://forum.openwrt.org/t/mediatek-mt7613e-5ghz-wifi-driver/73369/8.

Reply:
Yes, kmod-mt7615e includes MT7613 (MT7663) support and firmware is separated for each chip.